### ISSUE TITLE
Failed to parse source map

### ISSUE THREAD

**Issue by velislavatwone** [sarink/react-file-drop#91]
Using react-scripts 5.0.0 to bundle a JavaScript project and getting the following error:

```
WARNING in ./node_modules/react-file-drop/FileDrop.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '<full path>/node_modules/src/FileDrop.tsx' file: Error: ENOENT: no such file or directory, open '<full path>/node_modules/src/FileDrop.tsx'
```

Seems like it's caused by these changes https://github.com/sarink/react-file-drop/compare/v3.0.1...v3.0.3

Similar problems have been happening with other packages https://github.com/facebook/create-react-app/discussions/11767

**Comment by max-mathieu** [comment]
Facing a similar issue here. Just a warning though, but this is the only package we use that throws this warning

**Comment by sarink** [comment]
Can you share a test repo that demonstrates this issue? I don't really understand what could be causing it 🤔

**Comment by jay-herrera** [comment]
We are also seeing this same issue, version 3.1.5

**Comment by andrehen** [comment]
Yeah... I have the same problem:
`Failed to parse source map from '/home/andrehen/development/react-file-tree-visualizer/node_modules/src/FileDrop.tsx' file: Error: ENOENT: no such file or directory, open '/home/andrehen/development/react-file-tree-visualizer/node_modules/src/FileDrop.tsx'`

It's trying to locate the file in a directory that really doesn't exist inside node_modules: `/node_modules/src/FileDrop.tsx`

**Comment by josdejong** [comment]
I have exactly the same issue. The sourcemap `FileDrop.js.map` contains a reference to a source file that is not in the npm package:

```json
"sources":["../src/FileDrop.tsx"]
```

**Comment by josdejong** [comment]
I had the same issue in an other project of mine where I use rollup to generate a bundle. To solve the issue there I had to set `sourceMap` and `inlineSources` to `true` in the configuration options for the typescript plugin:

```js
// rollup.config.js
// ...
export default {
  // ...
  output: [
    {
      // ...
      sourcemap: true
    }
  ],
  plugins: [
    // ...
    typescript({ sourceMap: true, inlineSources: true }),
    // ...
  ]
}
```

This project has a different setup but maybe it gives a clue on how to fix the sourcemap

**Comment by sarink** [comment]
Does anyone have an example repo?

**Comment by bcepuran** [comment]
@sarink here is a repo you can review: https://github.com/bcepuran/react-file-drop-map-error

if you `npm install` and `npm start` you should see the warning in the terminal window.

**Comment by bcepuran** [comment]
I also submitted a PR that I think will address the issue by turning off the creation of the source map

**Comment by sarink** [comment]
Fixed by #96
